index.html

```diff
@@ -7,13 +7,103 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" type="text/css" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
-<body>
+<body class="container">
+            <header class="row">
+                <h3>Project 1: A small bussiness </h3>
```

Comment:
Rows should only contain elements with a `col-*` attribute. Wrap your title in a `col-*` element, e.g.:

```html
<header class="row">
  <div class="col-3">
    <h3>Project: A Small Business</h3>
  </div>
  <nav class="col-9">
    ...
```